index.html

```diff
@@ -346,6 +345,47 @@ <h2>Skills</h2>
 
         </div>
     </div>
+  
+  
+  
+    <footer>
+        <div>
+            <p id="footer-text">
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+        </div>
+        <form action="./index.html">
+            <label>
+                <input type="text" name="user" placeholder="Full name">
```

Comment:
- [ ] Kindly make sure the name input has `maxlength="30"` attribute as specified in the design.